### DISCUSSION TITLE
Auto generating (watching) Scss to CSS does not live update Five Server

### DISCUSSION THREAD

**Discussion by scarecrow2k6** [yandeu/five-server#20] [Help / Questions]
Don't know if this is a Five Server issue or I am missing something. Working on CSS (with auto save) updates fine with Five Server but when I work on Scss and it generates CSS, browser needs to be manually refreshed. Any solution to this? Thanks.

Using:
Five Server v. 0.0.31 and Live Sass Compiler v. 5.0.4 (by Glenn Marks).

**Comment by yandeu** [comment]
Try to ignore scss files.
See: https://github.com/yandeu/five-server#watch--ignore

**Reply by scarecrow2k6** [reply]
Hi Yannick, thanks for your reply. Just to be sure, are you advising not to use sass or scss or change some kind of setting? Thanks. 

ps I have just started learning, so I am new to all of this so please elaborate.

**Reply by yandeu** [reply]
You can use sass and scss.

Since "Live Sass Compiler" is watching your scss files, you probably don't want "Five Server" to also watch the scss files at the same time.

I have just tested it and it works. Can you please try the simple setup below?

_Btw, make sure you don't use auto-save with VSCode._

---

My setup:

```html
// index.html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Document</title>
    <link rel="stylesheet" href="style.css" />
  </head>
  <body>
    <h1>Title</h1>
  </body>
</html>
```
```scss
// style.scss
* {
  background-color: rgb(211, 81, 81);
}

h1 {
  font-size: 48px;
  color: white;
}
```
```js
// fiveserver.config.js
module.exports = {
  ignore: /\.s[ac]ss$/i
}
````

If you want instant updates, use the file below instead.

```js
// fiveserver.config.js
module.exports = {
  ignore: /\.s[ac]ss$/i,
  injectBody: true
}
````

**Reply by simonl65** [reply]
@yandeu Thank you for maintaining this extension - it's GREAT!

I have the same issue as above, but your suggestion doesn't appear to work for me. I suspect it _may_ be because Live Sass Compiler (by Glenn Marks v5.1.1) first deletes the style.css file before re-adding it with the changes? Is Five Server seeing the file go missing but not it's return?

**Reply by simonl65** [reply]
Aha! I needed to add:
```injectCss: true```
